### ISSUE TITLE
Remove usage of exec from twisted/names/dns.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1799]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1799|
|Type|task|
|Created|2006-06-06 23:46:47Z|

exec is grotty.  plus it breaks pyflakes.

<details><summary>Searchable metadata</summary>

```
trac-id__1799 1799
type__task task
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__names names
keywords__ 
time__1149637607000000 1149637607000000
changetime__1150982057000000 1150982057000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

Ready for review in [branches/remove-exec-1799](branches/remove-exec-1799)

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set owner to @exarkun|
|-|-|

Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [17339]) Merge remove-exec-1799

Author: exarkun
Reviewer: radix
Fixes [#1799](https://github.com/twisted/twisted/issues/1799)

This replaces the use of exec statements to define module attributes
for various DNS protocol constants with explicit assignments.  It also
removes a redundant import of the warnings module.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
